.github/workflows/package.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.9'
+          python-version: '3.12'
```

Comment:
I would stick with `3.9` since I don't see any advantage in updating it at this point.